src/MetricsMsg.chpl

```diff
@@ -14,7 +14,8 @@ module MetricsMsg {
 
     use ArkoudaMapCompat;
 
-    enum MetricCategory{ALL,NUM_REQUESTS,RESPONSE_TIME,AVG_RESPONSE_TIME,SYSTEM,SERVER,SERVER_INFO};
+    enum MetricCategory{ALL,NUM_REQUESTS,RESPONSE_TIME,AVG_RESPONSE_TIME,TOTAL_RESPONSE_TIME,
+                        TOTAL_MEMORY_USED,SYSTEM,SERVER,SERVER_INFO};
```

Comment:
Not really an issue, but do we need the new line here?

Reply:
Really? I pulled it down and showed that was as well. No worries! should not hurt anything!